helm/distributed-idol/templates/dih/_container.tpl

```diff
@@ -0,0 +1,58 @@
+# BEGIN COPYRIGHT NOTICE
+# Copyright 2024 Open Text.
+#
+# The only warranties for products and services of Open Text and its affiliates and licensors
+# ("Open Text") are as may be set forth in the express warranty statements accompanying such
+# products and services. Nothing herein should be construed as constituting an additional warranty.
+# Open Text shall not be liable for technical or editorial errors or omissions contained herein.
+# The information contained herein is subject to change without notice.
+#
+# END COPYRIGHT NOTICE
+
+{{- define "distributedidol.dih.container.base" -}}
+{{- $root := get . "root" | required "distributedidol.dih.container.base: missing root" -}}
+{{- $component := get . "component" | required "distributedidol.dih.container.base: missing component" -}}
+{{/* specific merges to container specification */}}
+command: ["/bin/sh"]
+args:
+  - -c
+  - |
+    cd /dih
+    ln -sf /opt/idol/dih/data ./data
+    if [ -e /opt/idol/dih/data/dih.cfg ]
+    then
+      echo Using existing dih.cfg
+    else
+      cp /opt/idol/dih/data/dih.install.cfg /opt/idol/dih/data/dih.cfg
+    fi
+    ln -sf /opt/idol/dih/data/dih.cfg ./dih.cfg
+    ./run_idol.sh
+lifecycle:
+  preStop:
+    exec:
+      command:
+        - /bin/bash
+        - "-c"
+        - |
+{{ tpl ($root.Files.Get "resources/dih_preStop.sh") $root | nindent 16 }}
+{{- if $component.envConfigMap }}
+envFrom:
+  - configMapRef: {{ $component.envConfigMap | quote }}
+{{ end }}
```

Comment:
Incorrect use of `configMapRef`
```suggestion
envFrom:
  - configMapRef: 
      name: {{ $component.envConfigMap | quote }}
{{ end }}
```

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/80/commits/9a1ac167e1cb22914a236785d6c2b31fb9833922

Reply:
Need to delete the `{{- if $component.envConfigMap }}` above

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/80/commits/66b249b6f3f30f253e14220e6919b16c9e5f3bd5